WordPress/gutenberg
# Block Library: Remove client side variation fallbacks

## What?
PR removes client-side variation fallbacks for Navigation Link and Template Parts blocks.

Currently cannot fully remove "variation enhancers," since they handle the icon rendering. I think we can only use Dashicons when registering variations on the server side.

## Why?
Like inline comments suggest the fallback was only needed while the Gutenberg plugin supported WP < 5.8.

## Testing Instructions
1. Open a Post or Page.
2. Run snippet below in DevTools console and confirm variations are correctly loaded.
3. Manually check Navigation Link and Template Parts blocks.

## Snippet

```js
wp.blocks.getBlockType('core/template-part').variations
wp.blocks.getBlockType('core/navigation-link').variations
```
